zubhub_frontend/zubhub/src/views/activity_details/MakeActivity.js

```diff
@@ -0,0 +1,126 @@
+        <Typography className={classes.textStyle}>
+          {activity.introduction ? <div dangerouslySetInnerHTML={{ __html: activity.introduction }} /> : ''}
+        </Typography>
+        {activity.images?.length > 0 && (
+          <div className={classes.imageStyle}>
+            {activity.images.map((img, index) => (
+              <img
+                key={index}
+                src={img.image?.file_url}
+                alt={`Image ${index}`}
+                className={classes.imageIndividualStyle}
+              />
+            ))}
+          </div>
+        )}
+        <Box className={classes.verticalSpace}></Box>
+
+        <Typography className={`${classes.pdfSubtitle}`}>Category</Typography>
+        <div className={clsx(commonClasses.displayFlex, commonClasses.gap, commonClasses.flexWrap)}>
+          {activity.category?.map((cat, index) => (
+            <Pill key={index} text={cat} />
+          ))}
+        </div>
+        <Box className={classes.verticalSpace}></Box>
+
+        <Typography className={`${classes.pdfSubtitle}`}>Class Grade</Typography>
+        <div className={clsx(commonClasses.displayFlex, commonClasses.gap, commonClasses.flexWrap)}>
+          <Pill text={activity.class_grade} />
+        </div>
+        <Box className={classes.verticalSpace}></Box>
+
+        <Typography className={`${classes.pdfSubtitle}`}>Materials Used</Typography>
+        <Typography
+          className={`${classes.textStyle} ${classes.customTypographyStyle}`}
+          theme={'bubble'}
+          readOnly={true}
+          value={activity.materials_used || ''}
+        >
+          {activity.materials_used ? <div dangerouslySetInnerHTML={{ __html: activity.materials_used }} /> : ''}
+        </Typography>
+        {console.log(activity.materials_used_image)}
+        <div className={classes.imageStyle}>
+          <img
+            src={activity.materials_used_image?.file_url}
+            // alt={`Image ${index}`}
+            className={classes.imageIndividualStyle}
+          />
+        </div>
+
+        {/* {activity.materials_used_image && <Gallery images={[activity.materials_used_image?.file_url]} />} */}
+
+        <br></br>
+
+        {activity.making_steps?.map((step, index) => (
+          <Collapsible key={index} title={`Step ${step?.step_order}: ${step.title}`}>
+            {step.image?.length > 0 && <Gallery images={step.image?.map(img => img?.file_url)} />}
+            {step.description && (
+              <ReactQuill
+                className={classes.textStyle}
+                theme={'bubble'}
+                readOnly={true}
+                value={step.description || ''}
+              />
+            )}
+          </Collapsible>
+        ))}
+        <Box className={classes.verticalSpace}></Box>
+      </Container>
+      <Container className={classes.footerStyle}>
+        <Container>
+          <img className={classes.pdfFooterLogoStyle} src={images.unstructured} alt={'Faridah_ux'} />
```

Comment:
the alt text is hardcoded. Is that supposed to be (asking a genuine question here because I have no idea)?
Is it is supposed to be hardcoded then we need to translate it on the frontend